e2e/__tests__/detectOpenHandles.ts

```diff
@@ -69,6 +69,21 @@ it('does not report promises', () => {
   expect(textAfterTest).toBe('');
 });
 
+const nodeMajorVersion = Number(process.versions.node.split('.')[0]);
+
+if (nodeMajorVersion >= 11) {
```

Comment:
Maybe `nodeMajorVersion >= 11 ? it : it.skip` so that it shows up as a skipped test case on Node < 11 instead of not at all. Seems cleaner to me, but not sure if we have precedent for conditionally skipping a test _case_ in the codebase - @SimenB?

Reply:
We have methods for skipping tests on windows or jasmine/circus. Adding some `skipOnNodeVersions()` that takes a conditional or someone might make sense? That logic is in https://github.com/facebook/jest/blob/master/packages/test-utils/src/ConditionalTest.ts. Works by wrapping the tests you might wanna skip in a `describe` and calling one of the functions